avalanche/benchmarks/scenarios/rl_scenario.py

```diff
@@ -9,14 +9,108 @@
 # Website: avalanche.continualai.org                                           #
 ################################################################################
 """Reinforcement Learning scenario definitions."""
-from typing import Union, Sequence
+from avalanche.benchmarks.scenarios import CLExperience, ExperienceAttribute, CLScenario, EagerCLStream
+from typing import Callable, List, Union, Dict
+import numpy as np
+import torch
+import random
 
 try:
-    from gym import Env
+    from gym import Env, Wrapper
 except ImportError:
-    # empty class to make sure everything below works without changes
+    # empty classes to make sure everything below works without changes
     class Env:
         pass
+    class Wrapper:
+        pass
+
+
+class RLExperience(CLExperience):
+
+    def __init__(self, env: Union[Env, Callable[[], Env]], n_envs: int = 1, task_label: int = None, current_experience: int = None, origin_stream=None):
+        # current experience and origin stream are set when iterating a CLStream by default
+        super().__init__(current_experience, origin_stream)
+        self.env = env
+        self.n_envs = n_envs
+        # task label to be (optionally) used for training purposes
```

Comment:
I think we can drop the requirement of having mandatory task labels (eventually set to `None`).

Also, in a multi-task scenario you should also set `use_in_eval=True`

Reply:
This is still a problem, it should be `self.task_label = ExperienceAttribute(task_label, use_in_train=True, use_in_eval=True)`. If `task_labels=None` is you can also decide to not having the attribute or having  `self.task_label = ExperienceAttribute(task_label)`